genie-web/src/test/groovy/com/netflix/genie/web/rpc/grpc/services/impl/v4/GrpcHeartBeatServiceImplSpec.groovy

```diff
@@ -20,13 +20,16 @@ package com.netflix.genie.web.rpc.grpc.services.impl.v4
 
 import com.netflix.genie.proto.AgentHeartBeat
 import com.netflix.genie.proto.ServerHeartBeat
+import com.netflix.genie.test.categories.UnitTest
 import com.netflix.genie.web.services.AgentRoutingService
 import io.grpc.stub.StreamObserver
+import org.junit.experimental.categories.Category
 import org.springframework.scheduling.TaskScheduler
 import spock.lang.Specification
 
 import java.util.concurrent.ScheduledFuture
 
+@Category(UnitTest)
```

Comment:
Long term I'd like to get rid of all this and move to either use JUnit 5 (which I think has different support) and/or use the [Nebula Facet Plugin](https://github.com/nebula-plugins/nebula-project-plugin) to provide clean separation of tests

Reply:
Right now it seems the only purpose is filtering when executing 2 specific gradle tasks that look at these.
I agree it's probably not worth the trouble.

Reply:
The facet plugin looks good!